libbeat/template/load.go

```diff
@@ -80,6 +82,43 @@ func (l *Loader) Load() error {
 	return nil
 }
 
+// Generate generates the template and writes it to a file based on the configuration
+// from `output_to_file`.
+func (l *Loader) Generate() error {
```

Comment:
We could probably now use this function also in https://github.com/elastic/beats/blob/master/dev-tools/cmd/index_template/index_template.go